core/context/context.go

```diff
@@ -12,3 +12,9 @@ const (
 	ExecModeSimulate
 	ExecModeFinalize
 )
+
+// TODO: remove
+type ContextKey string
+
+// TODO: remove
+const CometInfoKey ContextKey = "comet-info"
```

Comment:
For usage see: https://github.com/cosmos/cosmos-sdk/blob/kocu/genesis-v2/x/distribution/keeper/abci.go#L27
I think this can be resolved with https://github.com/cosmos/cosmos-sdk/pull/19602 which adds a message to retrieve `CometInfo` from the RouterService, or optionally including a keeper.